upstream-community-operators/hpe-csi-driver-operator/1.0.0/hpe-csi-driver-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,446 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: hpe-csi-driver-operator.v1.0.0
+  namespace: placeholder
+  annotations:
+    alm-examples: >-
+      [
+        {
+          "apiVersion": "storage.hpe.com/v1",
+          "kind": "HPECSIDriver",
+          "metadata": {
+            "name": "csi-driver",
+            "namespace": "hpe-csi"
+          },
+          "spec": {
+            "crd": {
+              "nodeInfo": {
+                "create": false
+              }
+            },
+            "cspName": "nimble-csp",
+            "flavor": "kubernetes",
+            "imagePullPolicy": "Always",
+            "images": {
+              "csiDriverImage": "hpestorage/csi-driver:v1.1.0",
+              "cspImage": "hpestorage/nimble-csp:v1.1.0"
+            },
+            "logLevel": "info",
+            "secret": {
+              "backend": "192.168.1.1",
+              "create": true,
+              "name": "nimble-secret",
+              "password": "admin",
+              "serviceName": "nimble-csp-svc",
+              "servicePort": "8080",
+              "username": "admin"
+            },
+            "storageClass": {
+              "allowVolumeExpansion": true,
+              "create": true,
+              "defaultClass": false,
+              "name": "hpe-standard",
+              "parameters": {
+                "accessProtocol": "iscsi",
+                "fsType": "xfs",
+                "volumeDescription": "Volume created by the HPE CSI Driver for Kubernetes"
+              }
+            }
+          }
+        }
+      ]
+    categories: Storage
+    certified: 'true'
+    createdAt: ''
+    description: >-
+      A Container Storage Interface (CSI) Driver for HPE Storage. The CSI driver
+      allows you to use HPE Storage with your preferred container orchestrator.
+    containerImage: 'hpestorage/csi-driver-operator:v1.0.0'
+    support: 'HPE'
+    capabilities: Basic Install
+    repository: 'https://github.com/hpe-storage/co-deployments/tree/master/operators/hpe-csi-operator'
+spec:
+  displayName: HPE CSI Driver Operator
+  description: >
+    The [HPE CSI Driver for
+    Kubernetes](https://github.com/hpe-storage/csi-driver) leverages HPE storage
+    platforms to provide scalable and persistent storage for stateful
+    applications. Please refer to our GitHub page for supported HPE platforms and full list of supported CSI features.
+
+    The HPE CSI Operator packages, deploys, manages, upgrades HPE CSI Driver on
+    Kubernetes and OpenShift for dynamic provisioning of persistent volumes on
+    HPE storage systems.
+
+    ## Platform and Software Dependencies
+
+    For platform dependencies for HPE CSI driver please refer to
+    [prerequisites](https://github.com/hpe-storage/co-deployments/tree/master/helm/charts/hpe-csi-driver#prerequisites)
+
+  maturity: beta
+  version: 1.0.0
+  replaces: ''
```

Comment:
Hi @shivamerla , please remove `replaces` as there is nothing to replace during the first commit. And pipeline can fail on the empty value

Reply:
@J0zi Done. Thanks. Also, since we already have Operator certified with RedHat and it appears in RedHat catalog, do i still need to submit one under community-operators folder as well?

Reply:
@J0zi Looks like OLM installation itself is failing with CI tests. Can you please check?